src/QueryParameters/GetProjectsParameters.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Spatie\FloatSdk\QueryParameters;
+
+use Spatie\FloatSdk\Concerns\HasPaginationAndSort;
+
+class GetProjectsParameters
+{
+    use HasPaginationAndSort;
+
+    /**
+     * @param  ?array<string>  $fields
+     * @param  ?array<string>  $expand
+     */
+    public function __construct(
+        public ?bool $active = null,
+        public ?string $projectCode = null,
+        public ?int $clientId = null,
+        public ?bool $nonBillable = null,
+        public ?string $tagName = null,
+        public ?string $modifiedSince = null,
+        public ?array $fields = null,
+        public ?array $expand = null,
+        ?int $page = null,
+        ?int $perPage = null,
+        ?string $sort = null,
```

Comment:
Are these really all nullable? I'd also add the project id here, or is that project code in Float?

Reply:
Yes, everything is optional: https://developer.float.com/api_reference.html#!/Projects/getProjects

I'd add a different GetProjectRequest to retrieve a single project.
https://developer.float.com/api_reference.html#!/Projects/getProject